test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -454,11 +514,14 @@ def forward(self, x):
         #    [convert_element_type_1, sub, mul_1, dequantize_per_channel, convolution]
         # 4. Quantization fusion in post-grad fusion pass * 2
         #    [qconv2d_pointwise_default, div_1, round_2, add_1, clamp_min_1, clamp_max_1, convert_element_type_2]
+        # 5. Qconv2d_add * 1
+        #    [qconv2d_pointwise_default_1, convert_element_type_5, sub_2, mul_5, add_3, mul_6, round_4, add_4,
+        #     clamp_min_3, clamp_max_3, convert_element_type_6]
         self._test_common(
             mod,
             (v,),
-            10,
-            40,
+            11,
+            51,
```

Comment:
In the future it'd be great if we could make the tests more modular. Checking exact # of matches across separate invocations make it harder to detect which part regressed if something fails

Reply:
Sure, thanks for the suggestions.